layer_factory/CMakeLists.txt

```diff
@@ -145,6 +145,21 @@ foreach(SUBDIR ${ST_SUBDIRS})
 
 endforeach()
 
+# Create the layer_factory_linux.json file
+if(UNIX)
+    set(TARGET_NAME layer_fractory_linux)
```

Comment:
typo: fractory
Is CI working w the typo?

Reply:
TARGET_NAME is a variable being set to "layer_fractory_linux". It is referenced layer via the variable, so no errors. It would probably be best to remove the variable and use the string in the add_custom_target command a few lines down.